app/templates/import_holidays.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+
+{% block content %}
+
+<form class="m-3" action="update_holidays" method="post" enctype="multipart/form-data">
+    <h5 class="title">Import holidays using ics file</h5>
+    <h6 class="title">Use the upload button to choose ics file</h6>
+    <a href="https://www.feiertagskalender.ch/export.php?geo=3542&hl=en#" target="_blank">Check this website to export holidays by country</a>
+    <div class="input-group">
+        <input type="file" class="form-control-sm" accept=".ics" name="file">
```

Comment:
Maybe it would be better to add a label here :)

Reply:
not sure i understand - what kind of label?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
I'm just not sure what is the label use case for this page.

Reply:
It improves a11y :)

Reply:
Thanks. good to know!